tests/integrated/test-env-vars.R

```diff
@@ -0,0 +1,132 @@
+# Temporarily skip these on newer Connect versions because the manifest for this
+# content specifies an old version of R not found in newer images.
+skip_if(safe_server_version(test_conn_1) > "2024.03.0")
+
+# ensure that RSPM is being used so these do not take eternity
+rmd_content <- deploy_example(test_conn_1, "rmarkdown")
```

Comment:
If we wanted to maintain these tests across the full set of tested versions, we could just duplicate the `rmarkdown` content with different manifests, and condition the content name on `safe_server_version(test_conn_1)`.

Reply:
I'll make a note of that as a possibility in the followup issue, an alternative to using rsconnect.